numpy/core/code_generators/ufunc_docstrings.py

```diff
@@ -32,6 +32,8 @@ def get(name):
             For other keyword-only arguments, see the
             :ref:`ufunc docs <ufuncs.kwargs>`.
     """).strip(),
+    'BROADCASTABLE': ("If ``x1.shape != x2.shape``, they must be broadcastable"
+                      " to a common shape."),
```

Comment:
Perhaps note that this common shape is the shape of the output.

Also, can you add `_2` to the end of this name to prevent us accidentally using it in 3-arg functions?